SwiftCompilerSources/Sources/SIL/Builder.swift

```diff
@@ -375,4 +375,9 @@ public struct Builder {
     let endMutation = bridged.createEndCOWMutation(instance.bridged, keepUnique)
     return notifyNew(endMutation.getAs(EndCOWMutationInst.self))
   }
+
+  public func createMarkDependence(value: Value, base: Value, _ escaping: MarkDependenceInst.Escaping) -> MarkDependenceInst {
```

Comment:
E.g. here: IMO, `createMarkDependence(value: Value, base: Value, isNonEscaping: Bool)` would be as readable as with the `Escaping` enum.